java/client/src/main/java/glide/api/commands/ConnectionCommands.java

```diff
@@ -0,0 +1,15 @@
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Connection Management Commands interface.
+ *
+ * @see: <a href="https://redis.io/commands/?group=connection">Server Management Commands</a>
+ */
+public interface ConnectionCommands {
+
+    CompletableFuture<String> ping();
```

Comment:
Move all javadocs that you have from `RedisClient` to the interfaces. The client(s) would inherit the javadoc.